specs-go/config.go

```diff
@@ -25,6 +25,8 @@ type Spec struct {
 	Linux *Linux `json:"linux,omitempty" platform:"linux"`
 	// Solaris is platform specific configuration for Solaris containers.
 	Solaris *Solaris `json:"solaris,omitempty" platform:"solaris"`
+	// Windows is platform specific configuration for Windows based containers, including Hyper-V containers.
+	Windows *Windows `json:"solaris,omitempty" platform:"windows"`
```

Comment:
Embarrassing copy/paste error! Yes, that one is definitely fixed in the upcoming push.